dbsync.py

```diff
@@ -595,7 +601,9 @@ def init(conn_cfg, mc, from_gpkg=True):
         # this is not a first run of db-sync init
         db_proj_info = _get_db_project_comment(conn, conn_cfg.base)
         if not db_proj_info:
-            raise DbSyncError("Base schema exists but missing which project it belongs to")
+            raise DbSyncError("Base schema exists but missing which project it belongs to. "
+                              f"Both schemata `{conn_cfg.base}` and `{conn_cfg.modified}` should be deleted in the database "
+                              "so that DB Sync can set up sync properly")
```

Comment:
```suggestion
                              f"This may be a result of a previously failed attempt to initialize DB sync. You can delete both schemas `{conn_cfg.base}` and `{conn_cfg.modified}` to fix this error and restart DB sync.")
```

(schemata may be more proper plural but it sounds too academic :slightly_smiling_face: ... also databases seem to use "schemas" as the plural... https://www.postgresql.org/docs/current/ddl-schemas.html)